src/Stratis.Bitcoin.IntegrationTests/ConsensusManagerTests.cs

```diff
@@ -21,20 +23,18 @@ public ConsensusManagerTests()
 
             Type consensusType = typeof(NBitcoin.Consensus);
             consensusType.GetProperty("MaxReorgLength").SetValue(this.network.Consensus, (uint)20);
-
-            this.network.Consensus.Options = new ConsensusOptionsTest();
         }
 
         private class ConsensusOptionsTest : PosConsensusOptions
         {
             public override int GetStakeMinConfirmations(int height, Network network)
             {
-                return height < 10 ? 5 : 10;
+                return height < 15 ? 5 : 20;
             }
         }
 
         [Fact]
-        public void ConsensusManager_Fork_Occurs_MinerNode_Reorgs_AndResyncs_ToBestHeight()
+        public void ConsensusManager_Fork_Occurs_Node_Reorgs_AndResyncs_ToBestHeight()
```

Comment:
Can you just specify in the comment the scenario this covers?

Reply:
@dangershony This very basic re-org test doesn't specifically belong to any of the scenarios in Ivan/Apro's test document. Is this OK? I will start work on those specific tests after this is merged 👍